src/views/run-list.js

```diff
@@ -0,0 +1,22 @@
+export default Backbone.View.extend({
+  tagName: 'ul',
+
+  initialize() {
+    this.render();
+
+    this.listenTo(this.collection, 'sync', this.render);
+  },
+
+  render() {
+    this.$el.html(this.template(this.collection));
+  },
+
+  template(collection) {
+    return this.collection.map((run) => {
+      return `
+      <li>${run.get('date')}
+      ${run.get('time')}</li>
+      <a href="#${run.id}"><i class="fa fa-plus"></a></li>`;
```

Comment:
The Plus icon should go to a new page, clicking on any part of the run itself will be a link to the details.